### DISCUSSION TITLE
Export for Android

### DISCUSSION THREAD

**Discussion by iRadEntertainment** [lennart-finke/GodoTeX#5] [Ideas]
At the moment the plugin doesn't work on Android (the Readme specifies that it's only for Windows and Linux).
But as far as I understood it may work if we can find a workaround for the missing `System.Drawing` since it is not implemented on Android (as this user was asking on a StackOverflow [post](https://stackoverflow.com/questions/34862868/how-to-add-system-drawing-in-xamarin).
I am not good at C# so I'm having a hard time importing the correct library that can run on Android. If I figure it out I will post it here

**Comment by zbyna** [comment]
System.Drawing.dll issue can be solved by adding this target  for android:
```xml
To target .NET Standard 2.0, add the following to your .csproj file:
<PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
<PropertyGroup>
```
and
```XML
 <PropertyGroup>
    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
 </PropertyGroup>
```

![image](https://github.com/user-attachments/assets/61f78698-708d-43b5-a8fa-9bb1ae837f1c)

netstandard.dll assembly is in Godot 3.5 export template for android.

But then issues described in https://github.com/lennart-finke/GodoTeX/issues/14 arises.
